consensus/src/chained_bft/consensus_types/block.rs

```diff
@@ -318,10 +326,20 @@ where
     }
 
     pub fn height(&self) -> Height {
+        // Height:
+        // - should not exceed std::u64::MAX - 1 to ensure the parent check doesn't
+        // cause addition overflow.
+        // (Block::make_block)
+        assumed_postcondition!(self.height < std::u64::MAX);
         self.height
     }
 
     pub fn round(&self) -> Round {
+        // Round numbers:
+        // - are reset to 0 periodically.
+        // - do not exceed std::u64::MAX - 2 per the 3 chain safety rule
+        // (BlockStore::need_sync_for_quorum_cert)
```

Comment:
Sorry I didn't look closely enough at what you're putting there. The correct ref is 
https://github.com/libra/libra/blob/b840e9393de925b4bf0205a95c5fec458d9ea5e6/consensus/src/chained_bft/safety/safety_rules.rs#L216-L218